.gitignore

```diff
@@ -14,6 +14,10 @@
 # Misc junk folder
 junk/
 
+# Helm dependency build generated files
+charts/*/charts
```

Comment:
Is this a decent way to handle this?

Reply:
Yeah, this is fine. I'm used to seeing it as something more like `charts/**/charts` which is more recursive than this, but this doesn't need to be more recursive cause `git` will handle more recursive cases based on the root path.

More concretely, I was worried that this glob would ignore `charts/foo/charts` but not `charts/foo/charts/bar/charts`, but since this glob matches `charts/foo/charts`, `git` also extends it to ignore `charts/foo/charts/bar/charts`.

Here's a little script I wrote to try it out:
```bash
mkdir -p charts/foo/charts/bar/charts/

touch charts/foo/Chart.yaml
touch charts/foo/charts/bar/Chart.yaml
touch charts/foo/charts/bar/charts/Chart.yaml

git add charts/foo

git status

On branch add-grafana-configuration
Your branch is up to date with 'origin/add-grafana-configuration'.

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
        new file:   charts/foo/Chart.yaml
```

Reply:
Cool thanks! Felt a little weird, so I was thinking there might be some reason it would be ill advised